cmd/server/main.go

```diff
@@ -107,6 +107,10 @@ type ProxyRunOptions struct {
 	authenticationAudience string
 	// Path to kubeconfig (used by kubernetes client)
 	kubeconfigPath string
+
+	// Proxy strategy used by the server, can be either "destHost" or "default".
+	// If not set, the "default" strategy will be used.
+	proxyStrategy string
```

Comment:
We haven't been great about this up to this point. However limited value variables should really be enums. Should consider making this an enum.

Reply:
will make it enum